tests/test_select_gen.py

```diff
@@ -10,8 +11,8 @@
 @needs_py39
 def test_select_gen() -> None:
     result = subprocess.run(
-        [sys.executable, "scripts/generate_select.py"],
-        env={"CHECK_JINJA": "1"},
+        [sys.executable, Path("scripts") / "generate_select.py"],
+        env={**os.environ, "CHECK_JINJA": "1"},
```

Comment:
The `env` stuff is basically the same fix as what was done in https://github.com/fastapi/sqlmodel/pull/969, but shortened.